### PR TITLE
Week-3-server: Step 4

### PR DESCRIPTION
**Pull request by rebekadittrich**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -42,6 +42,15 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.getWriter().println(convertGreetingsToJson());
   }
 
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    /* Get the input from the form. */
+    String greeting = request.getParameter("text-input");
```
Comment (pmueller-gg) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:49]: What happens when the parameter does not exist?

**Review by pmueller-gg — COMMENTED** [review]
One clarifying question.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (rebekadittrich) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:49]: I think it adds an empty string to the list of strings. I could add an if statement to check if it's empty before adding.

Comment (rebekadittrich) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:49]: I think it returns a null string which won't cause an issue but I will add a check for null regardless. Thank you!

**Review by pmueller-gg — APPROVED** [review]

**Pull request merged**